app/views/albums/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2> New Album </h2>
+  <%= form_for @album, action: 'create', method: :post do |f| %>
+<fieldset>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+    <br>
```

Comment:
I like that you added space to make your forms more readable. Aim for using CSS to affect the presentation, rather than a ```<br>``` in the HTML.